Winget-AutoUpdate/user-run.ps1

```diff
@@ -0,0 +1,131 @@
+<#
+.SYNOPSIS
+Handle user interaction from shortcuts and show a Toast
+
+.DESCRIPTION
+Act on shortcut run (DEFAULT: Check for updated Apps)
+
+.PARAMETER Logs
+Open the Log file from Winget-AutoUpdate installation location
+
+.PARAMETER Help
+Open the Web Help page
+https://github.com/Romanitho/Winget-AutoUpdate
+
+.EXAMPLE
+.\user-run.ps1 -Logs
+
+#>
+
+[CmdletBinding()]
+param(
+	[Parameter(Mandatory=$False)] [Switch] $Logs = $false,
+	[Parameter(Mandatory=$False)] [Switch] $Help = $false
+)
+
+function Show-Toast ($Title, $Message, $MessageType, $Balise, $OnClickAction) {
+
```

Comment:
We might use existing functions

Reply:
Then I need to rewrite the function and load just that (howto?)  into the **standalone** user-run.ps1...
...we always want to show the toast as a sign that something's happening when the user clicks the link regardless of NotificationLevel.

Reply:
...and initiate the log system etc...

Reply:
`. "$PSScriptRoot\functions\Start-NotifTask.ps1"`  (Dot Source Notation) - but then there's so much more to do - better user-run.ps1 as a **standalone**!